tiny-java-containers/jwebserver/Dockerfile.distroless-java-base.jlink

```diff
@@ -1,5 +1,21 @@
+FROM container-registry.oracle.com/graalvm/jdk:23 AS build
+RUN microdnf install -y binutils
```

Comment:
Why do we need to install `binutils`?

Reply:
Must have been needed earlier and not now.  Removed.